server.crt

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Remove these two files; or, if _absolutely_ needed for tests, put them into a `testdata` folder. However, I don't think they are needed. If so, maybe put them into a const like you did with the others.